packages/node_modules/@webex/webex-core/test/integration/spec/services/services.js

```diff
@@ -905,6 +915,47 @@ describe('webex-core', () => {
           assert.isAbove(Object.keys(unauthServices.list(false, 'signin')).length, 0);
           assert.equal(Object.keys(unauthServices.list(false, 'postauth')).length, 0);
         }));
+
+      const getActivationRequest = (requestStub) => {
```

Comment:
Can you move either, at the beginning of the describe block, inside the `beforeEach` or inside the new test (e.g. `it` block)? Just not in the middle of the block

Reply:
I left it just above the tests that use it for context (defining close to the only places it is used), but I have moved it to the start of the describe (after the constants, but before the `beforeEach`). Is there a general style in terms of organising tests I can refer to in future?

Reply:
The convention is to add it at the beginning of the describe block. If there are test that have common setups, we usually do a inner describe to group related-tests together